src/hoi4_world/countries/hoi4_countries_converter.cpp

```diff
@@ -88,7 +86,7 @@ std::map<std::string, hoi4::Country> hoi4::ConvertCountries(const std::map<int,
 
       std::optional<Country> new_country = ConvertCountry(source_country,
           source_country_technologies,
-          source_cultures,
+          source_world.GetCultureDefinitions(),
```

Comment:
I think vic3::Country should store a reference to World, so we can just call country::GetCulture() and it'll find it automatically

Reply:
Yeah, dependency injection cleans up function calls a lot.

Reply:
I'd be curious to see how this would play out via a quick PR. My initial impression is that it'd end up copying CultureDefinitions quite a bit, or passing references to them to distant functions. But I could easily be wrong.